students/chelsea_nayan/lesson05/database.py

```diff
@@ -0,0 +1,166 @@
+'''Adds products in a csv file to a MongoDB database'''
+# lesson 05: Consuming APIs
+
+# Checklist:
+# Create a product db with attributes that reflect the contents of the csv file
+# Import all data in the csv files into your MongoDB implementation
+# Write queries to retrieve the product database
+# Write a query to integrate customer and product data
+# no errors from pylint
+# quantity_available of '0' is understood as 'not available'
+# Create a test_database.py
+
+# pylint: disable=too-many-statements
+# pylint: disable=invalid-name
+# pylint: disable=too-many-locals
+
+import logging
+import csv
+import os
+from pymongo import MongoClient
+# Set up logger
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+class MongoDBConnection():
+    '''MongoDBConnection'''
+
+    def __init__(self, host='127.0.0.1', port=27101):
```

Comment:
This is not necessarily an issue, as your system may have a different setup. I use port 27017 on my laptop, otherwise the run will timeout (since port 27101 is blocked).